lib/index.js

```diff
@@ -138,6 +138,7 @@ var opts = exports.options = {
   , curve_secretkey: zmq.ZMQ_CURVE_SECRETKEY
   , curve_serverkey: zmq.ZMQ_CURVE_SERVERKEY
   , zap_domain: zmq.ZMQ_ZAP_DOMAIN
+  , immediate: zmq.ZMQ_DELAY_ATTACH_ON_CONNECT
```

Comment:
@khaldoun1, don't issue another PR, you're almost there! 

I think you may want to add `zmq.ZMQ_IMMEDIATE` based on the appropriate result from `zmq.version()`. 

For consistency I would also try to address this discrepancy in versions inside `longOptions` as well: https://github.com/khaldoun1/zeromq.node/blob/ZMQ_IMMEDIATE_Option/lib/index.js#L83